californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
When I build e.g. the californium-proxy module, then the resulting jar's manifest contains the following:

```
Manifest-Version: 1.0
Bnd-LastModified: 1490261180536
Build-Jdk: 1.8.0_121
Built-By: hak8fe
Bundle-Description: Cross-proxy module
Bundle-DocURL: http://www.eclipse.org/
Bundle-License: http://www.eclipse.org/org/documents/epl-v10.php, http:/
 /www.eclipse.org/org/documents/edl-v10.php
Bundle-ManifestVersion: 2
Bundle-Name: Californium (Cf) Proxy
Bundle-SymbolicName: californium-proxy
Bundle-Vendor: Eclipse Foundation
Bundle-Version: 2.0.0.SNAPSHOT
Created-By: Apache Maven Bundle Plugin
Export-Package: org.eclipse.californium.proxy;uses:="org.apache.http,org
 .eclipse.californium.core,org.eclipse.californium.core.coap,org.eclipse
 .californium.core.network,org.eclipse.californium.proxy.resources";vers
 ion="2.0.0",org.eclipse.californium.proxy.resources;uses:="com.google.c
 ommon.cache,org.eclipse.californium.core,org.eclipse.californium.core.c
 oap,org.eclipse.californium.core.network,org.eclipse.californium.core.s
 erver.resources";version="2.0.0"
Import-Package: com.google.common.cache;version="[15.0,16)",com.google.c
 ommon.collect;version="[15.0,16)",com.google.common.io;version="[15.0,1
 6)",com.google.common.primitives;version="[15.0,16)",org.apache.http,or
 g.apache.http.client,org.apache.http.client.protocol,org.apache.http.co
 nn,org.apache.http.entity,org.apache.http.impl,org.apache.http.impl.cli
 ent,org.apache.http.impl.nio,org.apache.http.impl.nio.reactor,org.apach
 e.http.message,org.apache.http.nio,org.apache.http.nio.protocol,org.apa
 che.http.nio.reactor,org.apache.http.params,org.apache.http.protocol,or
 g.apache.http.util,org.eclipse.californium.core;version="[2.0,3)",org.e
 clipse.californium.core.coap;version="[2.0,3)",org.eclipse.californium.
 core.network;version="[2.0,3)",org.eclipse.californium.core.network.con
 fig;version="[2.0,3)",org.eclipse.californium.core.server.resources;ver
 sion="[2.0,3)",org.eclipse.californium.proxy,org.eclipse.californium.pr
 oxy.resources
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
Tool: Bnd-3.0.0.201509101326
```
The imports for the Apache HttpComponents do not have any version information attached which will probably be a problem during resolution on a runtime, wouldn't it? In addition to that the HttpComponents artifacts do not seem to be OSGi bundles themselves, at least they are lacking the corresponding meta information in their manifests ...